operations/research-and-development/product/product-planning/how-to-use-productboard.md

```diff
@@ -113,113 +114,113 @@ Two very important notes regarding this integration:
 
-As long as you are logged into ProductBoard, the connectivity to the Mattermost insights board is secure and seamless.
+As long as you are logged into productboard, the connectivity to the Mattermost insights board is secure and seamless.
 
 ![](../../../../.gitbook/assets/productboard-chrome-extension.png)
 
-**Pro tip**: If you keep your call notes in a browser app (such as Google Docs), you can highlight the sections related to feedback and quickly send them over to ProductBoard with the extension.
+**Pro tip:** If you keep your call notes in a browser app (such as Google Docs), you can highlight the sections related to feedback and quickly send them over to productboard with the extension.
 
-## 3 - Day in the life of a Dev Lead
+## 3 A day in the life of a Dev Lead
 
-### 3a - Viewing the master feature list and customer insights
+### 3a Viewing the Master Feature list and customer insights
 
-As a **Contributor** in ProductBoard, you have access to view the [master feature list](https://mattermost.productboard.com/feature-board/1097524-master-feature-list). This is the view where the Product team organizes and prioritizes new features and functionality that is being considered or planned for Mattermost. You can search and browse all features and ideas and even read the notes and portal cards from PMs for a detailed description of the feature/idea. In the case of planned features, you will have complete insight into the status of the feature and a rough idea of where it sits on the roadmap. Although only PMs are able to modify the `Effort` estimate, it is highly encouraged for you to review and provide your feedback to your PM as well!
+As a **Contributor** in productboard, you have access to view the [Master Feature list](https://mattermost.productboard.com/feature-board/1097524-master-feature-list). This is the view where the Product team organizes and prioritizes new features and functionality that is being considered or planned for Mattermost. You can search and browse all features and ideas and even read the notes and portal cards from Product Managers for a detailed description of the feature/idea. In the case of planned features, you'll have complete insight into the status of the feature and a rough idea of where it sits on the roadmap. Although only Product Managers are able to modify the **Effort** estimate, it's highly encouraged for you to review and provide your feedback to your Product Manager as well!
 
-Within the detailed view of individual features, you can view the full list of insights added by other PMs, CSMs, SAs, or Support Managers.
+Within the detailed view of individual features, you can view the full list of insights added by other Product Managers, Customer Success Managers, Solutions Architects, or Support Managers.
```

Comment:
Confirming that these internal roles are accurate or whether they need to be updated/reviewed.

Reply:
see above